### ISSUE TITLE
Piranha.Extend.Blocks.HtmlColumnBlock - Piranha Version 11.0

### ISSUE THREAD

**Issue by hedronn** [PiranhaCMS/piranha.core#2050]
Whilst upgrading from 10 -> 11.0 compiler error - Piranha.Extend.Blocks.HtmlColumnBlock not in Namespace, has the HtmlColumnBlock been deprecated in v11.0?

**Comment by tidyui** [comment]
Yes it was replaced by the ColumnBlock many versions ago. If you need it you should be able to copy it into your application and register it manually in your application startup!

https://github.com/PiranhaCMS/piranha.core/blob/v10.4/core/Piranha/Extend/Blocks/HtmlColumnBlock.cs
